.github/workflows/master-snapshot-release.yml

```diff
@@ -1,4 +1,8 @@
 name: Test & Release Snapshot to Maven Central
+
+env:
+  MAVEN_ARGS: -V -ntp -Dhttp.keepAlive=false -e
```

Comment:
What's the reason for using these args specifically?

Reply:
To have better troubleshooting info:
-V is about dumping information about maven and java
-ntp is to avoid printing the download of artifacts
-e is about having some better error message if maven is failing

-Dhttp.keepAlive=false is a copy and paste but was used to fix some issue we had sometimes with the way the GitHub Actions environment is set up (I can remove it)

Reply:
`-Dhttp.keepAlive=false` removed